2023-07-25 14:35:32,420 INFO    MainThread:145613 [wandb_setup.py:_flush():76] Current SDK version is 0.15.5
2023-07-25 14:35:32,420 INFO    MainThread:145613 [wandb_setup.py:_flush():76] Configure stats pid to 145613
2023-07-25 14:35:32,420 INFO    MainThread:145613 [wandb_setup.py:_flush():76] Loading settings from /home/robot-lab/.config/wandb/settings
2023-07-25 14:35:32,420 INFO    MainThread:145613 [wandb_setup.py:_flush():76] Loading settings from /home/robot-lab/Documents/wandb/settings
2023-07-25 14:35:32,420 INFO    MainThread:145613 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-07-25 14:35:32,420 INFO    MainThread:145613 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-07-25 14:35:32,420 INFO    MainThread:145613 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-07-25 14:35:32,420 INFO    MainThread:145613 [wandb_init.py:_log_setup():507] Logging user logs to /home/robot-lab/Documents/wandb/run-20230725_143532-8934ibyk/logs/debug.log
2023-07-25 14:35:32,420 INFO    MainThread:145613 [wandb_init.py:_log_setup():508] Logging internal logs to /home/robot-lab/Documents/wandb/run-20230725_143532-8934ibyk/logs/debug-internal.log
2023-07-25 14:35:32,420 INFO    MainThread:145613 [wandb_init.py:init():547] calling init triggers
2023-07-25 14:35:32,420 INFO    MainThread:145613 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-07-25 14:35:32,421 INFO    MainThread:145613 [wandb_init.py:init():571] re-initializing run, found existing run on stack: d9901wi9
2023-07-25 14:35:32,422 INFO    MainThread:145613 [wandb_run.py:_finish():1887] finishing run paul-nieuwerburgh/BVAE/d9901wi9
2023-07-25 14:35:32,478 INFO    MainThread:145613 [jupyter.py:save_history():445] not saving jupyter history
2023-07-25 14:35:32,478 INFO    MainThread:145613 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-07-25 14:35:32,478 INFO    MainThread:145613 [wandb_init.py:_jupyter_teardown():435] cleaning up jupyter logic
2023-07-25 14:35:32,478 INFO    MainThread:145613 [wandb_run.py:_atexit_cleanup():2121] got exitcode: 0
2023-07-25 14:35:32,479 INFO    MainThread:145613 [wandb_run.py:_restore():2104] restore
2023-07-25 14:35:32,479 INFO    MainThread:145613 [wandb_run.py:_restore():2110] restore done
2023-07-25 14:36:40,421 INFO    MainThread:145613 [wandb_init.py:init():596] starting backend
2023-07-25 14:36:40,421 INFO    MainThread:145613 [wandb_init.py:init():600] setting up manager
2023-07-25 14:36:40,423 INFO    MainThread:145613 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-07-25 14:36:40,424 INFO    MainThread:145613 [wandb_init.py:init():606] backend started and connected
2023-07-25 14:36:40,428 INFO    MainThread:145613 [wandb_run.py:_label_probe_notebook():1233] probe notebook
2023-07-25 14:36:40,428 INFO    MainThread:145613 [wandb_run.py:_label_probe_notebook():1243] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-07-25 14:36:40,429 INFO    MainThread:145613 [wandb_init.py:init():705] updated telemetry
2023-07-25 14:36:40,429 INFO    MainThread:145613 [wandb_init.py:init():738] communicating run to backend with 60.0 second timeout
